src/pages/Login.tsx

```diff
@@ -62,9 +65,14 @@ export const Login = () => {
                 Sign in
               </Button>
             </Stack>
-            <Text>{loginMutation.isError ? (loginMutation as any).error.message : ''}</Text>
+            <Text color={'red'}>{loginMutation.isError ? (loginMutation as any).error.message : ''}</Text>
```

Comment:
"... as any" I don't like but if there's a need then I can live with this.

Reply:
:(